packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -454,6 +458,10 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			0x385 /* For clarity and paranoid defense against adding future attachment states */,
 		);
 
+		if (signal?.aborted) {
+			throw new Error("aborted before uploading");
```

Comment:
we should clean up the error handling here. something like:
```
function createAbortError(pending?){
return new LoggingError("uploadBlob aborted", {acked: pending.acked, status: pending.status};
```

then the until test should inspect the properties, rather than a unique string per case, as the strings cost bundle size